test/plugins/cookie/cookies.get.test.js

```diff
@@ -0,0 +1,59 @@
+import sinon from 'sinon';
+
+import CookiesPlugin from '../../../src/plugins/cookies';
+import state from '../data/cookie-state.json';
+import { create } from '../../../src/chrome-api';
```

Comment:
I think, there should be no spaces because of code style. The "Yandex" style declares that objects literals (including destructuring)  should not have spaces around the content.